### DISCUSSION TITLE
大家来分享下使用调试的小技巧吧

### DISCUSSION THREAD

**Discussion by hui-shao** [WankkoRee/WebViewPP#12] [General]
作者开发的这个模块简直神器，为了让模块发挥更大的作用（相互学习），大家来讨论下自己常用的一些操作吧。

-----

我先来一个：

**xx大学习快速完成：**
首先确认视频是用 `video` 标签载入，在控制台输入下面的代码就行，原理是设置进度到视频长度的前一秒。

```
var video = document.getElementsByTagName("video")[0];video.currentTime = video.duration-1
```

**Comment by WankkoRee** [comment]
哈哈哈就知道会有人用`vConsole`跳这个，以前老版本微信能直接开启`vConsole`就没我这个模块的事了。

另，如果需要隐藏`vConsole`的话（比如说截图等展示屏幕的情况），可以执行以下语句：
```javascript
vConsole.hideSwitch()
```

**Reply by hui-shao** [reply]
啊哈 感谢分享 不过这东西隐藏了要重新打开是不是只能 刷新网页/重新进入 了
Σ(ŎдŎ|||)ﾉﾉ

**Reply by WankkoRee** [reply]
也不一定，可以加一条定时任务去实现重新显示，比如说留10s用来截图，10s后自动重新打开。

```javascript
vConsole.hideSwitch();setTimeout(function(){vConsole.showSwitch()},1000*10);
```
